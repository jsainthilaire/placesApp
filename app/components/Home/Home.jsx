import React from 'react';
import { centeredContainer, title, description } from 'sharedStyles/styles.css'

const Home = () => (
  <section className={centeredContainer}>
    <h1 className={title}>Places</h1>
    <p className={description}>Save your places to visit and rock the world!</p>
  </section>
)

export default Home
