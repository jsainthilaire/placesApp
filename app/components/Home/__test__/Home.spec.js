import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import Home from '../Home'

test('Home should render correctly', () => {
  const wrapper = shallow(<Home />)
  expect(toJson(wrapper)).toMatchSnapshot()
})

