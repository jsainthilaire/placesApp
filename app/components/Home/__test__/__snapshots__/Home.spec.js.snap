// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home should render correctly 1`] = `
<section
  className="centeredContainer"
>
  <h1
    className="title"
  >
    Places
  </h1>
  <p
    className="description"
  >
    Save your places to visit and rock the world!
  </p>
</section>
`;
